### PR TITLE
Feature/IRs

### PR DESCRIPTION
**Pull request by JanSto1995**
fine-tuned IRs again, mostly addressing unnatural fade-outs..

### CODE REVIEW

**Comment by JanSto1995** [comment]
note that the Impulse Responses (IRs) are expecting their respective IR .wav file to be found in /var/lib/jacktrip/impulses/IRs_080422/

File: links/ConvReverbLink/config.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
             "src": "irPath",
             "label": "Space",
             "type": "string",
-            "defaultVal": "/var/lib/jacktrip/impulses/IMreverbs/French 18th Century Salon.wav",
+            "defaultVal": "/var/lib/jacktrip/impulses/080422/Reverb - Royal Hall.wav",
```
Comment (nwang92) [inline links/ConvReverbLink/config.json:10]: Typo in the path name here

**Comment by nwang92** [comment]
Changed the target merge branch from `main` to `develop`

**Comment by mikedickey** [comment]
Where do these impulse response files come from?

Looks like this already merged to develop, but we should not remove any of our existing impulse responses. It's fine to add new ones, but removing the existing ones would be a big step backwards.

**Comment by JanSto1995** [comment]
The impulse responses are 1) ones I've created/recorded myself 2) some of the previous open-source IRs, 3) measurements of my favorite reverbs. I've talked to Andrew about copyright concerns, happy to talk more.

I think we could simply keep the two previous folders with the IRs (called "IMreverbs" [https://www.voxengo.com/impulses/] and "EchoThiefImpulseResponseLibrary" [https://files.jacktrip.org/contrib/EchoThiefImpulseResponseLibrary.zip]).

**Comment by JanSto1995** [comment]
putting this on hold until we've received legal advice regarding the use of the impulse responses

**Pull request closed without merging**